### PR TITLE
[#144] edo2net node service

### LINKED ISSUE #144: Add predefined edo2net systemd service for tezos-node
# Clarification and motivation
In v8.2 edo protocol was updated, a bit later new testnet called edo2net was started. 
However, `tezos-node` doesn't have a predefined `edo2net` `--network` option. This makes
testnet usage quite inconvenient with v8.2 binaries because one has to manually provide node config for
edo2net.

Note that after #108 and #143, we should have `tezos-node` service with a custom config
and machinery for package post-installation scripts. 
I propose the following way to support edo2net: define one more `tezos-node` service.
Create `edo2net` node config file during `tezos-node` postinstallation stage.

# Acceptance criteria
`tezos-node` native packages have predefined systemd service for running edo2net node.

### PR DESCRIPTION
**Pull request by rvem**
## Description
Problem: v8.2 tezos-node doesn't have a predefined network for edo2net,
which makes it really inconvenient to run a node that uses this new
testnet.

Solution: Add a tezos-node-edo2net service, which uses edo2net config
created during the package installation.

~Currently based on #143~

## Related issue(s)

Resolves #144

#### Related changes (conditional)

- [ ] I checked whether I should update the [README](../tree/master/README.md)

- [ ] I checked whether native packaging works, i.e. native binary packages
  can be successfully built.

#### Stylistic guide (mandatory)

- [ ] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).

### CODE REVIEW

File: docker/package/packages.py
Comment (pasqu4le) [inline docker/package/packages.py:118]: where is this `network` coming from? :thinking:

**Review by pasqu4le — COMMENTED** [review]
mostly LGTM, will review again after all the dependencies are merged

File: docker/package/packages.py
Comment (rvem) [reply docker/package/packages.py:118]: I assume the last value from the `for` loop above is used here :see_no_evil:

**Comment by rvem** [comment]
@pasqu4le, all dependencies were merged, PR was rebased, please rereview

File: protocols.json
Comment (pasqu4le) [inline protocols.json:18]: Nitpick, but I suppose they should be in the opposite order.

File: docker/package/packages.py
Comment (pasqu4le) [inline docker/package/packages.py:137]: Shouldn't we embed `007` now as well (in the `optional_opam_deps` one line below)

**Review by pasqu4le — COMMENTED** [review]
It looks like there are some tests on `007` that are now failing.

File: docker/package/packages.py
Comment (rvem) [reply docker/package/packages.py:137]: For now, 007 is not an optional dependency, see [here](https://opam.ocaml.org/packages/tezos-node/tezos-node.8.2/)

**Comment by rvem** [comment]
Tests should be fixed now

**Review by pasqu4le — APPROVED** [review]
LGTM, one last little comment is to replace `Deactive` with `Deactivate` in the third commit message.

**Pull request merged**
